packages/pages/src/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import classNames from "classnames";
+import { BreadcrumbsProps } from "./types.js";
+import { Link } from "../link/link.js";
+import { cta } from "../link/methods.js";
+import "./breadcrumbs.css";
+
+export interface BreadcrumbsPropsDefault extends BreadcrumbsProps {
+  breadcrumbs: Array<[string, string]>;
```

Comment:
Why transform this to a more generic [string, string] instead of having them strongly typed by name?